robot/EDA/resources/AffiliationsPageObject.py

```diff
@@ -140,20 +133,6 @@ def Go_to_affiliations_edit_mode(self, loc):
         self.selenium.click_element(affiliations_locators["button_save_affiliation"])
         self.eda.close_toast_message()
 
-    def _expandShadowRoot(self, element):
```

Comment:
please add documentation

Reply:
please ignore, I was looking at wrong version, since this is deleted no documentation needed